src/app/zap-templates/partials/idl/cluster_attribute.zapt

```diff
@@ -0,0 +1,26 @@
+  {{! ensure indent}}{{#unless isWritable~}}
+  readonly {{!marker to place a space even with whitespace removal~}}
+{{~/unless~}}
+{{~!TODO: write only attributes should also be supported~}}
+{{~!TODO: non-reportable (nosubscribe) attributes should also be supported ~}}
```

Comment:
This is supported using `isReportable` and `isReportableAttribute` under zcl_attributes_server

Reply:
isReportable stopped working when I was using zcl_attributes_server. It was only working for the chip_* version.